static/js/lib/validation.js

```diff
@@ -78,11 +79,13 @@ export const legalAddressValidation = yup.object().shape({
       .string()
       .label("First Name")
       .trim()
+      .matches(NAME_REGEX, "Invalid First Name")
```

Comment:
We should probably have tests for all of the stuff in here, but could you add some at least for this regex validation?

Reply:
Test Added